### PR TITLE
Add OpenBSD port and build instructions in the README.md

### PR DESCRIPTION
**Pull request by hboetes**
This is the OpenBSD port as submitted to the ports team:

  https://marc.info/?l=openbsd-ports&m=161310210623977&w=2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +123,17 @@ Unstable macOS installers are generated after every commit to the master branch,
 
 - [Download Installer](https://github.com/Nicotine-Plus/nicotine-plus/actions?query=branch%3Amaster+event%3Apush+is%3Asuccess+workflow%3A%22Packaging%22)
 
+## OpenBSD
```
Comment (mathiascode) [inline README.md:126]: This should be under the `GNU/Linux, *BSD` section, before `Other Distributions`.

**Comment by mathiascode** [comment]
I'm willing to keep this in the main repo until the port is approved, as long as you maintain it for future Nicotine+ releases (if the port approval takes that long).

**Comment by hboetes** [comment]
Sure, I'll keep this up to date. Since it takes a bit of time for everyone to get the code checked into the ports tree and the packages I kindly request keeping this directory around for one year until after the import.

File: README.md [hunk 2]
```diff
@@ -123,6 +123,17 @@ Unstable macOS installers are generated after every commit to the master branch,
...
+
+This port has been submitted to the OpenBSD ports team and is awaiting approval. Here is how you can build and install the port on systems up to 6.9.
+
+```
+mkdir -p /usr/ports/mystuff/net
```
Comment (mathiascode) [inline README.md:131]: Information about cloning the git repo is missing.

Comment (hboetes) [reply README.md:132]: Yes, you are right. I've added a few more lines.

File: README.md (hunk 1 above)
Comment (mathiascode) [reply README.md:126]: Ping?

**Pull request closed without merging**